pyproject.toml

```diff
@@ -96,16 +96,19 @@ test = [
     "boltons",
     "scanpy",
     "httpx", # For data downloading
-    "dask[array,distributed]>=2022.09.2,<2024.8.0",
+    "dask[distributed]>=2022.09.2",
```

Comment:
The constraint is already there through `anndata[dask]`, no need to add it again.

```suggestion
    "dask[distributed]",
```

Reply:
Are `distributed` and `array` linked?

Reply:
i.e.I think you can install different versions of each?  Just tested this.

Reply:
Hmm, no `dask[array]` is _not_ a separate package, ok.

Reply:
Or rather, `distributed` is a separate package, but hooks back to `dask`

Reply:
Hence the confusion

Reply:
`dask[distributed]>=2022.09.2` means “install dask >=2022.09.2 with the extra `distributed` active”